### PR TITLE
Add morgan for logging server requests

### PR DESCRIPTION
**Pull request by rogerhutchings**
- Uses express to run the local server
- Adds morgan to log requests to stdout

This is the first of some changes I'm making to enhance logging in the front end apps, using the content app as a test bed. If any of the reviewers have any advice on what to actually _do_ with the request logs (new relic? graylog?), I'd love suggestions; otherwise, we can just leave it for production debugging.

cc #174

### CODE REVIEW

File: packages/app-content-pages/server/server.js [hunk 1]
```diff
@@ -2,24 +2,31 @@ if (process.env.NEWRELIC_LICENSE_KEY) {
...
-  const server = new http.Server((req, res) =>
-    configureNextServer(app, req, res))
+  const server = express()
+  server.use(morgan('combined'))
 
-  server.listen(port, err => {
-    if (err) {
-      throw err
-    }
+  server.get('*', (req, res) => {
```
Comment (camallen) [inline packages/app-content-pages/server/server.js]: Minor not blocking: Does this server ever have to handle other HTTP verbs? e.g. OPTIONS for cors?

Comment (rogerhutchings) [reply packages/app-content-pages/server/server.js]: Not afaik...

**Comment by rogerhutchings** [comment]
@camallen is the centralised logging already in place?

I've also come across an improvement; logging to stdout for http codes < 400, and stderr for 400 <= (via https://github.com/expressjs/morgan/issues/134#issuecomment-277254355)

**Comment by camallen** [comment]
> @camallen is the centralised logging already in place?
> 
> I'm logging everything to stdout at the moment, but I've also come across an improvement:
> 
> * stdout for http codes < 400
> * stderr for http codes >= 400
> 
> (via [expressjs/morgan#134 (comment)](https://github.com/expressjs/morgan/issues/134#issuecomment-277254355))

I am not sure about central logging, one for Adam via email / slack for when he's back in the office. `kubectl get logs $pod_name` works just fine for your deployed pods...

**Pull request merged**